rs/ic_os/os_tools/hostos_tool/src/main.rs

```diff
@@ -23,12 +21,12 @@ pub enum Commands {
         output_directory: String,
     },
     GenerateIpv6Address {
-        #[arg(short, long, default_value = "HostOS")]
-        node_type: String,
+        #[arg(short, long, default_value = "0")]
+        node_type: u8,
```

Comment:
Does it have a benefit if we pass numbers? It'd be nice to pass strings from the CLI for readability.